src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -49,7 +50,12 @@ export function scrollToFirstError() {
       document.body.scrollTop ||
       0;
     const position = errorEl.getBoundingClientRect().top + currentPosition;
-    Scroll.animateScroll.scrollTo(position - 10, getScrollOptions());
+    // Don't animate the scrolling if there is an open modal on the page. This
+    // prevents the page behind the modal from scrolling if there is an error in
+    // modal's form.
+    if (!body.classList.contains('modal-open')) {
+      Scroll.animateScroll.scrollTo(position - 10, getScrollOptions());
+    }
```

Comment:
Hm, I wonder if this would be a good candidate for the design system's Modal React component?

Reply:
How so?

Reply:
Nvm, I got mixed up with how sometimes tabbing in a modal can cause the background to scroll. Disregard!

Reply:
this change isn't even supposed to be part of this PR so I'm rebasing to remove it!